docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,22 +231,17 @@ V&V Checks:
     -    
   * - 3
     - % of each facility type have cesarian section capabilities
-    - Public/governmental health facility (62%), private for-profit health facility (60%), at home (0%), private not-for-profit health facility (86%), other (10%)
+    - At home (0%), in hospital(74%), in clinic/low-level facility (53%)
```

Comment:
Hmm, this is a bit surprising -- we had been assuming C-sections would only be available in high-level facilities.

Reply:
The EmONC data value is actually not explicitly for c-section capabilities, but whether the facility had an up-to-date and complete operating theatre, so that is another compelling reason to switch to the SARA data source here. My apologies for not realizing this earlier!

Reply:
Ok, interesting, thanks for looking into this. Yeah, I guess these are all preliminary numbers, so we just have to remember to update everything based on the best data once we have more stuff figured out.